server/main.py

```diff
@@ -31,6 +31,7 @@ class Payload(BaseModel):
     path_length: float
     simulate_slit: bool
     mode: str
+    databank:str
```

Comment:
Actually, something like this would restrict the input even more so I think this would be better:

```suggestion
    databank:  Literal["hitran", "gesia"]
```